webcompat/templates/web_modules/dropdown.html

```diff
@@ -1,3 +0,0 @@
-{% macro dropdown(config) %}
```

Comment:
Unfortunately this macro had to go -- since it was included at html render time by the jinja2 template parser. Now, with pre-compiled templates, that's way too late. The good news is the macro was longer than the end result -- it's just a single line div. Not a huge loss. :)